tests/manager/manager-var-dump-001.phpt

```diff
@@ -45,10 +45,10 @@ object(MongoDB\Driver\Manager)#%d (%d) {
       ["is_passive"]=>
       bool(false)
       ["tags"]=>
-      array(0) {
+      object(stdClass)#%d (0) {
```

Comment:
@bjori: This and the `last_is_master` field below are stdClass instances because `php_phongo_server_to_zval()` invokes `bson_to_zval()` on some of its BSON documents. I assume this is a debug handler where we definitely want to dump things as arrays, correct?

If so, perhaps this happens often enough that I should just create an alternative constant to `PHONGO_BSON_STATE_INITIALIZER` intended for this purpose.